doc/how_to/components/index.md

```diff
@@ -33,3 +35,66 @@ How to access and set widget values.
 How to add or remove components from a panel.
 :::
 ::::
+
+## Style Components
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} Apply CSS
```

Comment:
If we can find icons octicon icons for all of these that would be fabulous: https://sphinx-design.readthedocs.io/en/latest/badges_buttons.html#octicon-icons